### PR TITLE
Support c8s

### PR DESCRIPTION
**Pull request by phracek**
This pull request adds support for building, and testing image on c8s.
It also supports pushing this image to quay.io

### CODE REVIEW

**Comment by phracek** [comment]
[test-all]

**Comment by phracek** [comment]
[test-all]

**Comment by phracek** [comment]
[test-all]

File: .github/workflows/build-and-push.yml [hunk 1]
```diff
@@ -40,3 +49,4 @@ jobs:
           docker_context: ${{ matrix.dockerfile_path }}
           dockerfile_path: ${{ matrix.dockerfile_path }}
           tag: ${{ matrix.tag }}
+          suffix: ${{ matrix.suffix }}
```
Comment (zmiklank) [inline .github/workflows/build-and-push.yml:52]:
```suggestion
          suffix: ${{ matrix.tag }}
```
It seems, that tag and suffix are the same.

**Comment by zmiklank** [comment]
Thanks for PR, I just found one small issue in build-and-push (see above). Else LGTM.

File: .github/workflows/build-and-push.yml (hunk 1 above)
Comment (phracek) [reply .github/workflows/build-and-push.yml:52]: No please not. When we will support Fedora build-and-push actions, suffixes will not be used at all.

**Pull request merged**